azure-pipelines.yml

```diff
@@ -35,6 +35,7 @@ stages:
               export AZURE_SEARCH_WORD_FUZZINESS=1
               export GOOGLE_GTM_CONTAINER_ID=GTM-WJ5TW34
               export GOOGLE_TRACKING_ID=UA-6838115-11
+              GOOGLE_USE_DEBUG=true
```

Comment:
Should have `export `?

Reply:
Also, do we want debug to be on in production?

Reply:
Yes, yes it should have `export` - good spot. This pipeline is release to dev environment when merged to master.